src/components/Page/base.css

```diff
@@ -38,14 +38,29 @@ body {
   color: var(--color-black);
 }
 
+/* Focus state styles for interactive elements */
+
+button:focus,
+input:focus,
+select:focus,
+textarea:focus,
+a:focus {
+  outline: 2px dotted var(--color-gray-hover);
+  outline-offset: 5px;
+}
+
+body:not(.user-is-tabbing) button:focus,
+body:not(.user-is-tabbing) input:focus,
+body:not(.user-is-tabbing) select:focus,
+body:not(.user-is-tabbing) textarea:focus,
+body:not(.user-is-tabbing) a:focus {
+  outline: none;
+}
+
 /* Prevent mobile browsers from changing font-size */
 * {
   -webkit-text-size-adjust: none;
   text-size-adjust: none;
-
-  &:focus {
-    outline: 0;
-  }
 }
 
 /* TODO: check it later */
```

Comment:
@pavelgrinchenko do you remember what is this? do we have a ticket for this?

Reply:
It was in [the old styles](https://github.com/iterative/dvc.org/blob/03c2355df50b10a1bedec2f442da4b97e5d6252c/src/styles.js#L31-L33) and I only transferred it carefully to the new version to not break anything.

Reply:
@pavelgrinchenko I'm specifically curious about "/* TODO: check it later */" part. It's not clear what to check.

Reply:
Ah, got it. I thought about migration this code to `src/components/Documentation/Markdown/styles.module.css`.

Reply:
@pavelgrinchenko so, is it still relevant? could you create a ticket with some explanation please?

Reply:
(and remove this TODO from code)

Reply:
Created PR for this one https://github.com/iterative/dvc.org/pull/1171

Reply:
thanks @pavelgrinchenko ! 🙏